### PR TITLE
[15.0][MIG] project_template_milestone: Migration to 15.0.

### PR DESCRIPTION
**Pull request by CoraVanga**

### CODE REVIEW

File: project_template_milestone/models/project_project.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Patrick Wilson <patrickraymondwilson@gmail.com>
```
Comment (quannguyen1560) [inline project_template_milestone/models/project_project.py]: I think this file is duplicated.

Comment (CoraVanga) [reply project_template_milestone/models/project_project.py]: I removed the file

**Comment by quannguyen1560** [comment]
@DinhTien24 can you fix the pipeline please?

**Comment by CoraVanga** [comment]
> @DinhTien24 can you fix the pipeline please?

@quannguyen1560 it is fixed now

**Review by quannguyen1560 — APPROVED** [review]
LGTM

File: project_template_milestone/README.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+* Patrick Wilson
+* Open Source Integrators
+
+Contributors
+~~~~~~~~~~~~
+
+* Patrick Wilson <patrickraymondwilson@gmail.com>
+* Open Source Integrators
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
```
Comment (hieulucky111) [inline project_template_milestone/README.rst:66]: put your name here.

File: project_template_milestone/models/project.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+from odoo import models
+
+
+class ProjectTemplate(models.Model):
+    _inherit = "project.project"
+
+    def create_project_from_template(self):
+        self.ensure_one()
+        res = super().create_project_from_template()
+        project = self.env["project.project"].browse(res["res_id"])
```
Comment (hieulucky111) [inline project_template_milestone/models/project.py]: project = self.browse(res["res_id"])

File: project_template_milestone/readme/CONTRIBUTORS.rst [hunk 4]
```diff
@@ -0,0 +1,3 @@
+* Patrick Wilson <patrickraymondwilson@gmail.com>
+* Open Source Integrators
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
```
Comment (hieulucky111) [inline project_template_milestone/readme/CONTRIBUTORS.rst:3]: put your name here

**Review by hieulucky111 — CHANGES_REQUESTED** [review]

**Review by hieulucky111 — APPROVED** [review]
Code Review
LGTM

**Pull request merged**